src/RpcClient/RpcClient.cs

```diff
@@ -196,83 +192,89 @@ public static ContractState ContractStateFromJson(JObject json)
+            var result = await RpcSendAsync("getrawtransaction", txHash).ConfigureAwait(false);
+            return result.AsString();
         }
 
         /// <summary>
         /// Returns the corresponding transaction information, based on the specified hash value.
         /// verbose = true
         /// </summary>
-        public RpcTransaction GetRawTransaction(string txHash)
+        public async Task<RpcTransaction> GetRawTransactionAsync(string txHash)
         {
-            return RpcTransaction.FromJson(RpcSend("getrawtransaction", txHash, true));
+            var result = await RpcSendAsync("getrawtransaction", txHash, true).ConfigureAwait(false);
+            return RpcTransaction.FromJson(result);
         }
 
         /// <summary>
         /// Calculate network fee
         /// </summary>
         /// <param name="tx">Transaction</param>
         /// <returns>NetworkFee</returns>
-        public long CalculateNetworkFee(Transaction tx)
+        public async Task<long> CalculateNetworkFeeAsync(Transaction tx)
         {
-            var json = RpcSend("calculatenetworkfee", Convert.ToBase64String(tx.ToArray()));
+            var json = await RpcSendAsync("calculatenetworkfee", Convert.ToBase64String(tx.ToArray()))
+                .ConfigureAwait(false);
             return (long)json["networkfee"].AsNumber();
         }
 
         /// <summary>
         /// Returns the stored value, according to the contract script hash (or Id) and the stored key.
         /// </summary>
-        public string GetStorage(string scriptHashOrId, string key)
+        public async Task<string> GetStorageAsync(string scriptHashOrId, string key)
         {
-            if (int.TryParse(scriptHashOrId, out int id))
-            {
-                return RpcSend("getstorage", id, key).AsString();
-            }
-
-            return RpcSend("getstorage", scriptHashOrId, key).AsString();
+            var result = int.TryParse(scriptHashOrId, out int id)
+                ? await RpcSendAsync("getstorage", id, key).ConfigureAwait(false)
+                : await RpcSendAsync("getstorage", scriptHashOrId, key).ConfigureAwait(false);
+            return result.AsString();
         }
 
         /// <summary>
         /// Returns the block index in which the transaction is found.
         /// </summary>
-        public uint GetTransactionHeight(string txHash)
+        public async Task<uint> GetTransactionHeightAsync(string txHash)
         {
-            return uint.Parse(RpcSend("gettransactionheight", txHash).AsString());
+            var result = await RpcSendAsync("gettransactionheight", txHash).ConfigureAwait(false);
+            return uint.Parse(result.AsString());
         }
 
         /// <summary>
         /// Returns the current NEO consensus nodes information and voting status.
         /// </summary>
-        public RpcValidator[] GetValidators()
+        public async Task<RpcValidator[]> GetValidatorsAsync()
```

Comment:
Can do, but it would be nice to get this PR merged and then make updates to master than keep having to adjust this PR to other changes. I'm going on vacation later this week, so I won't be able to keep adapting this PR in advance of preview 4